pkg/core/object/replicate.go

```diff
@@ -0,0 +1,68 @@
+package object
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+)
+
+const (
+	validInterval  = 10 // in epoches
+	currentVersion = 6  // it is also a number of fields
+
+	cidKey        = "cid"
+	oidKey        = "oid"
+	sizeKey       = "size"
+	deletedKey    = "deleted"
+	lockedKey     = "locked"
+	validUntilKey = "validuntil"
+)
+
+// MetaInfo uses NEO's map (strict order) serialized format as a raw
+// representation of object's meta information.
+//
+// This (ordered) format is used (keys are strings):
+//
+//	"cid": _raw_ container ID (32 bytes)
+//	"oid": _raw_ object ID (32 bytes)
+//	"size": payload size
+//	"deleted": array of _raw_ object IDs
+//	"locked": array of _raw_ object IDs
+//	"validuntil": last valid epoch number for meta information
+//
```

Comment:
In some sense `network` is already present in `cid` the way it's derived now, but I'm still not sure about potential replays. Also, we could use this index for large objects, so some chain data can be useful.

Reply:
ok, added network magic